docs/en/datasets/detect/coco12-formats.md

```
@@ -0,0 +1,234 @@
+| JPEG   | `.jpeg`   | JPEG with full extension             | Train     |
+| JPG    | `.jpg`    | JPEG with short extension            | Train     |
+| JP2    | `.jp2`    | JPEG 2000 - medical/geospatial       | Val       |
+| MPO    | `.mpo`    | Multi-Picture Object (stereo images) | Val       |
+| PNG    | `.png`    | Portable Network Graphics            | Val       |
+| TIF    | `.tif`    | TIFF with short extension            | Val       |
+| TIFF   | `.tiff`   | Tagged Image File Format             | Val       |
+| WebP   | `.webp`   | Modern web image format              | Val       |
+
+## Dataset Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)